docs/usage.md

```diff
@@ -9,8 +9,6 @@
   * [Reproducibility](#reproducibility)
 * [Main arguments](#main-arguments)
   * [`--sample_sheet`](#--sample_sheet)
```

Comment:
There is a kind of convention in `nf-core` now, that the input will be given with the parameter `--input`

Reply:
ok docs are updated